src/main/java/jenkins/scm/api/SCMHeadOrigin.java

```diff
@@ -43,6 +45,7 @@
  *
  * @since 2.2.0
  */
+@ExportedBean
```

Comment:
Note that #35 exported `SCMHead.origin` yet neglected to add this annotation, causing

    org.kohsuke.stapler.export.NotExportableException: class jenkins.scm.api.SCMHeadOrigin$Fork doesn't have @ExportedBean